repository-data/application/src/main/resources/hcm-config/namespaces/website/publishedworkchapter.yaml

```diff
@@ -222,115 +247,40 @@ definitions:
             hipposysedit:validators:
             - required
             jcr:primaryType: hipposysedit:field
-          hipposysedit:node: true
```

Comment:
There seems to be quite a lot of deleted stuff in this file. Did you mean to delete stuff?

Reply:
> There seems to be quite a lot of deleted stuff in this file. Did you mean to delete stuff?

The Bloomreach CMS was on auto export, therefore if anything was deleted it would have been deleted by Bloomreach. Nothing should have/ or was deleted manually. However, I did check to see if the published work chapter works as expected and it passed the criteria.